xbmc/Application.cpp

```diff
@@ -211,7 +211,7 @@
 #if defined(TARGET_ANDROID)
 #include "platform/android/activity/XBMCApp.h"
 #include "platform/android/activity/AndroidFeatures.h"
-#include "platform/android/jni/Build.h"
+#include "androidjni/Build.h"
```

Comment:
I always use `<>` for includes that are not part of the project's source tree and `""` for includes part of the source tree. So I'd go for `<>` here.